persist-cli-tests/src/test/java/io/ballerina/persist/tools/utils/GeneratedSourcesTestUtils.java

```diff
@@ -52,7 +54,7 @@ public class GeneratedSourcesTestUtils {
      * Represents persist commands.
      */
     public enum Command {
-        INIT,
+        ADD,
```

Comment:
Are we removing the INIT command completely?

Reply:
No, we haven't removed the `INIT` command. It's just I converted all the `init` tests to the `add` command tests. I will add a test case for `init` as well.